### ISSUE TITLE
unzip.exe + OpenDLP.exe crash  

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kuza55/opendlp#71]
```
What steps will reproduce the problem?
1.Deploy agent to WinXP and/or Win7. Testing with three clients 2 are XP. All 
three scans fail.  
2.During scanning OpenDLP agent will crash. Percentage of completion various.  
3.

What is the expected output? What do you see instead?
Completion of scan but service crashes.  

What version of the product are you using? On what operating system?
0.5b

Please provide any additional information below.

* In all cases crashes occur during the process of running unzip.exe.  This may 
or may not be within the realm of issue 70.  

```

Original issue reported on code.google.com by `Smartbox...@gmail.com` on 18 Aug 2012 at 7:51

**Comment by GoogleCodeExporter** [comment]
```
Log file
```

Original comment by `Smartbox...@gmail.com` on 18 Aug 2012 at 7:53

Attachments:
- [log.txt](https://storage.googleapis.com/google-code-attachments/opendlp/issue-71/comment-1/log.txt)

**Comment by GoogleCodeExporter** [comment]
```
Can you attach the file "C:\Documents and Settings\bahl\Application 
Data\Adobe\Acrobat\10.0\rdrmessage.zip" to this bug report? Thanks.
```

Original comment by `andrew.O...@gmail.com` on 18 Aug 2012 at 1:46

**Comment by GoogleCodeExporter** [comment]
```
rdrmessage.zip
```

Original comment by `Smartbox...@gmail.com` on 18 Aug 2012 at 4:23

Attachments:
- [rdrmessage.zip](https://storage.googleapis.com/google-code-attachments/opendlp/issue-71/comment-3/rdrmessage.zip)

**Comment by GoogleCodeExporter** [comment]
```
Here is another one I found on the same workstation
```

Original comment by `Smartbox...@gmail.com` on 18 Aug 2012 at 4:25

Attachments:
- [log.txt](https://storage.googleapis.com/google-code-attachments/opendlp/issue-71/comment-4/log.txt)
- [comm.jar](https://storage.googleapis.com/google-code-attachments/opendlp/issue-71/comment-4/comm.jar)

**Comment by GoogleCodeExporter** [comment]
```
Thanks for attaching the files. While I look into this, could you re-scan those 
systems with the old agent executable from OpenDLP 0.4.4? We made some code 
changes to the OpenDLP 0.5 agent, and I would like to rule out (or rule in) 
these changes as the source of the problem.

I have attached the 0.4.4 binary here, but you can also grab it from a previous 
release. Just overwrite the 0.5 "/var/www/OpenDLP/bin/OpenDLPz.exe" with this 
one, thanks.
```

Original comment by `andrew.O...@gmail.com` on 18 Aug 2012 at 4:40

Attachments:
- [OpenDLPz.exe](https://storage.googleapis.com/google-code-attachments/opendlp/issue-71/comment-5/OpenDLPz.exe)

**Comment by GoogleCodeExporter** [comment]
```
The rdrmessage.zip and comm.jar file were both from the same WinXP x86 system.  
The next upload is from a Win7 64bit system 
```

Original comment by `Smartbox...@gmail.com` on 18 Aug 2012 at 4:44

Attachments:
- [log.txt](https://storage.googleapis.com/google-code-attachments/opendlp/issue-71/comment-6/log.txt)
- [Assets1_1.zip](https://storage.googleapis.com/google-code-attachments/opendlp/issue-71/comment-6/Assets1_1.zip)

**Comment by GoogleCodeExporter** [comment]
```

Update:
I replaced the OpenDLPz.exe file and scanned the problem file with success.  I 
am now performing a full scan.  
Thank you much
```

Original comment by `Smartbox...@gmail.com` on 18 Aug 2012 at 4:57

**Comment by GoogleCodeExporter** [comment]
```
Update II:
A full scan on the XP workstation in question completed with success. Replacing 
the OpenDLPz.exe file seemed to do the trick.  Let me know if I can be of 
further assistance in testing any fix or update.  

Thanks again for your time.

BJ
```

Original comment by `Smartbox...@gmail.com` on 18 Aug 2012 at 6:22

**Comment by GoogleCodeExporter** [comment]
```
This has been fixed in OpenDLP 0.5.1, which was just released today. I reverted 
to the OpenDLP 0.4.4 binary that was attached to this bug report.

We will continue to look into what was causing this crash with the updated 
agent, but I thought this bug was important enough to require an OpenDLP 
update. Thanks for reporting it!
```

Original comment by `andrew.O...@gmail.com` on 28 Aug 2012 at 12:49
- Changed state: **Done**
